src/lib/CEPEngine_extensions.js

```diff
@@ -16,7 +16,7 @@
 //
 // Note: So far all native file i/o functions are synchronous, and aynchronous file i/o is TBD.
 
-/** Version v12.0.0 */
+/** Version v11.0.0 */
```

Comment:
how can it be that in these files only a version was changed in the comment and nothing else?